notebooks/official/ml_ops/stage1/get_started_dataflow.ipynb

```diff
@@ -0,0 +1,1220 @@
+{
```

Comment:
<p>I think we need to add a Open in Colab link</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/350/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>